### DISCUSSION TITLE
SIDKick with DAC issues with 250407 C64?

### DISCUSSION THREAD

**Discussion by tom-stock** [frntc/SIDKick-pico#92] [Q&A] [answered]
I just installed a sidkick DAC version, running only one channel (left) thru the C64 audio output.  Most audio sounds fine such as ghostbusters for example, but when I play SID tunes (regardless of which emulation) I have problems with missing notes, extra notes, chopping of audio, it just sounds quite bad.  I think it might be related to just one voice,  I've checked all solder joints, the only thing I can think of is that this is actually an issue with the mainboard not the SIDPico, caps or connections around the sid socket.

https://www.youtube.com/watch?v=tdIS498xv7U

I'm using very .21.   Thank you.

**Comment by frntc** [comment, marked as answer]
Sometimes the SID sockets are too loose, yes. If that's not the case: have you checked the bus timings? (https://github.com/frntc/SIDKick-pico#troubleshooting)

Caps on the board are not used (there are no pins on the SKpico PCB for that ;-))

**Reply by tom-stock** [reply]
Hi thanks so much for the reply! I did notice that mentioned but could not find any information about how to use it. I read a bit deeper and I understand now.  I'm on a mac so I'll have to dig out a windows machine to give it a try. Will report back! Thanks!

**Reply by frntc** [reply]
You can also try the small program that runs on the C64! (not guaranteed, but works most of the time)

**Reply by tom-stock** [reply]
I dropped the timing to 13 and 9, and it sounds perfect now!   I'm so excited to be able to change between 6581 and 8580 this is awesome. Thank you so much for the help.  I'll be updating that video.
